src/server/pps/server/s3g_sidecar.go

```diff
@@ -211,9 +211,12 @@ func (s *k8sServiceCreatingJobHandler) OnCreate(ctx context.Context, jobInfo *pp
 		return nm
 	}
 	selectorlabels := map[string]string{
-		"app":       ppsutil.PipelineRcName(jobInfo.Job.Pipeline.Name, jobInfo.PipelineVersion),
-		"suite":     "pachyderm",
-		"component": "worker",
+		// NOTE: this used to be ppsutil.PipelineRcName(jobInfo.Job.Pipeline.Name, jobInfo.PipelineVersion)
```

Comment:
Hmm, doesn't the selector need to match the worker pod labels? It seems like `app` would have to be `pipeline` here for that to work